tests/features/s3_credentials.feature

```diff
@@ -11,19 +11,19 @@ Feature: ch_s3_credentials tool
     Scenario Outline:: chadmin s3 check work correctly
     When we execute command on clickhouse01
     """
-    chadmin ch-s3-credentials check
+    ch-monitoring s3-credentials-config
     """
     Then we get response
     """
     0;OK
     """
     When we execute command on clickhouse01
     """
-    chadmin ch-s3-credentials --metadata-address=http_mock01:8080 update --endpoint=storage.com
+    chadmin s3-credentials-config --metadata-address=http_mock01:8080 update --endpoint=storage.com
     """
     And we execute command on clickhouse01
     """
-    chadmin ch-s3-credentials check --present
+    ch-monitoring s3-credentials-config --present
```

Comment:
lets also uncomment below. I made separate PR with such changes, but is looks like it can be part of this one.
https://github.com/yandex/ch-tools/pull/262 
```
 ## Commented because version `latest` will be greater than any other version.
    ##@require_version_24.11
    ##Examples:
    ##|header|
    ##|access_header|
    ##
    ## @require_version_less_than_24.11
```

Reply:
CI tests fail for the latest version on main branch. I'd postpone enabling additional checks until CI get fixed for ClickHouse 24.11.